fashrevwall/twitter_client/TwitterClient.py

```diff
@@ -60,7 +62,27 @@ def get_images_by_hashtag(self, hashtag, n):
                 print "This tweet doesn't contain an image."
                 continue
             try:
+                #in the future, for speeding up the process if number of tweets is extremely high: http://stackoverflow.com/questions/7943233/fast-way-to-discover-the-row-count-of-a-table
+                conn = psycopg2.connect(database = "fashrevwall", host="127.0.0.1", port="5432") #opens a conection to the DB
+                print "Opened database succesfully"
+                cur = conn.cursor() #creates a cursor which will be used to work on the database programming with Python
+                cur.execute('SELECT count(*) FROM wall_tweet')
+                row = cur.fetchone()
+                db_size = row[0]
+                print "Number of tweets in DB: %d \n" % db_size 
+
+                if db_size == 8:
+                    cur.execute('DELETE FROM wall_tweet WHERE ID = 7;') #delete last tweet --but IDs are not in order
```

Comment:
We shouldn't be looking at the ID but the created_at date. We need to order by date, then get the oldest and delete that tweet.